Doc/library/functools.rst

```diff
@@ -90,6 +91,15 @@ The :mod:`functools` module defines the following functions:
    differ in their keyword argument order and may have two separate cache
    entries.
 
+   If *user_function* is specified, it must be a callable. This allows the
+   *lru_cache* decorator to be applied directly to a user function, leaving
+   the *maxsize* at its default value of 128::
+
+       @lru_cache
+       def summary_statistics(data):
```

Comment:
It may be not so good example, since this function can be used only for immutable collections (tuples, but not lists, arrays or iterators).

Reply:
I rather liked this example, but will change it to another example that uses a typically immutable argument.